packages/fxa-payments-server/server/lib/amplitude.test.js

```diff
@@ -34,6 +34,7 @@ jest.mock('../config', () => ({
     switch (key) {
       case 'amplitude':
         return mockAmplitudeConfig;
+      default:
```

Comment:
Curious, what's this for?

Reply:
The linter had a warning that the switch needs a default statement. So this just adds it in to keep the linter happy.